aptos-move/aptos-gas/src/transaction.rs

```diff
@@ -92,6 +90,13 @@ impl StorageGasParameters {
             + num_bytes_create * self.per_byte_create
             + num_bytes_write * self.per_byte_write
     }
+
+    fn write_op_bytes(key_size: NumBytes, value_size: NumBytes) -> NumBytes {
+        let kb = NumBytes::new(1024);
+
+        let total = key_size + value_size;
+        total.checked_sub(kb).unwrap_or(NumBytes::zero())
```

Comment:
guess we need to feature gate here? Any recommendations? @vgao1996

Reply:
It seems like you are gating this by the gas feature version now, which looks good to me